### PR TITLE
Added Hate Speech Icon based on Threshold defined in Config

### PR DESCRIPTION
**Pull request by limmabean**
Could someone with the hate speech algorithm running tell me if this works?

### CODE REVIEW

**Review by harshilshah4251 — COMMENTED** [review]
Tested and works great.
image

File: config/secrets.json.example [hunk 1]
```diff
@@ -102,12 +102,17 @@
...
   "secret": "Replace this string when installing aggie with something else!!",
   "adminParty": false,
   "experiment": false,
   "experimentFile": "test/end-to-end/fixtures/experiment_reports.json",
   "adminPassword": "letmein1",
   "adminEmail": "aggie-admin@example.com",
-  "detectHateSpeech": false
+  "detectHateSpeech": false,
+  "hateSpeechThreshold": {
+      "threshold": 20e-2,
```
Comment (harshilshah4251) [inline config/secrets.json.example:114]: maybe add a comment there that this value needs to be between 0 and 1

Comment (cooperka) [reply config/secrets.json.example:114]: Agree this should be better documented, unfortunately JSON doesn't allow comments... the readme would be a good spot for this.

Is 20% thought to be a sensible default? Most server admins will accept it blindly.

File: config/secrets.json.example [hunk 2]
```diff
@@ -102,12 +102,17 @@
       "filename": "logs/analytics.log"
     }
   },
+
```
Comment (cooperka) [inline config/secrets.json.example]: Please remove

**Review by cooperka — APPROVED** [review]
Besides documentation, code looks great.

**Pull request merged**